cmd/notary/tuf.go

```diff
@@ -357,11 +360,23 @@ func (ps passwordStore) Basic(u *url.URL) (string, string) {
 }
 
 func getTransport(gun string, readOnly bool) http.RoundTripper {
+	// Attempt to get a root CA from the config file. Nil is the host defaults.
+	rootPool := x509.NewCertPool()
+	rootCAFile := viper.GetString("remote_server.root_ca")
+	if rootCAFile != "" {
+		rootCert, err := trustmanager.LoadCertFromFile(filepath.Join(configPath, rootCAFile))
```

Comment:
Non-blocking:  This does not work if an absolute path is passed - it will just join the configPath with the absolute path, which doesn't exist.  Probably if we wanted to support absolute paths as well as relative paths, there could be a `IsAbs` check first?

Otherwise, if we didn't want to support absolute paths, we should probably that the config file only supports paths relative to the config path.